Makefile

```diff
@@ -14,7 +14,8 @@ export GOPROXY
 
 PKGS=$(shell go list ./... | grep -v -E '/vendor/|/test|/examples')
 GOLANG_FILES:=$(shell find . -name \*.go -print)
-ASSETS=$(shell grep -oh '[^"]*/.*\.yaml' pkg/manifests/manifests.go | sed 's/^/assets\//')
+ASSETS=$(shell grep -oh '[^"]*/.*\.yaml' pkg/manifests/manifests.go \
+          | grep -v '%.*yaml' | sed 's/^/assets\//')
```

Comment:
`-v %.yaml` is need because `%s-policy.yaml` is used in manifest.go and that isn't a valid asset.